Deployment/Kubernetes/TensorRT-LLM_Autoscaling_and_Load_Balancing/containers/server.py

```diff
@@ -212,66 +212,6 @@ def execute_triton(args):
     exit(result)
 
 
-# ---
-
-
-def initialize_model(args):
```

Comment:
with `initialize_model` removed, how is the model getting generated?

Reply:
The model generation is now the responsibilty of the user and should be done before they start the service.
Steps for the same are outlined here https://github.com/triton-inference-server/tutorials/tree/ibhosale_update/Deployment/Kubernetes/TensorRT-LLM_Autoscaling_and_Load_Balancing#model-preparation-steps